app/models/comment.rb

```diff
@@ -0,0 +1,3 @@
+class Comment < ApplicationRecord
+  belongs_to :user
```

Comment:
- [ ] It would be better if the Comment model has a method that `updates the comments counter for a post`